Livecheckables/docbook-xsl.rb

```diff
@@ -0,0 +1,4 @@
+class DocbookXsl
+  livecheck :url   => "https://github.com/docbook/xslt10-stylesheets.git",
+            :regex => %r{^release\/(\d+(?:\.\d+)+)$}i
```

Comment:
```suggestion
            :regex => %r{^(?:release/)?(\d+(?:\.\d+)+)$}i
```

As before, we don't need to escape forward slashes when using the `%r{}` regex literal syntax.

Similar to the other PRs, we could make `release/` optional here, to match tags like `1.2.3` and `release/1.2.3`, in case release tags ever move away from the `release/` prefix. Snapshot versions have a different prefix (`snapshot/`) and also use a YYYY-MM-DD format, so we don't have to worry about erroneously picking them up.